tailwind.config.js

```diff
@@ -51,6 +54,7 @@ module.exports = {
       fontFamily: {
         body: ['Inter', 'system-ui', 'sans-serif'],
         heading: ['Manrope', 'sans-serif'],
+        sans: ['Inter', ...defaultTheme.fontFamily.sans],
```

Comment:
please note this font style applied to the whole site as default, let me know if we need to have different fonts

Reply:
That's desirable yep